### DISCUSSION TITLE
DarkThemeToggle icon size

### DISCUSSION THREAD

**Discussion by idc77** [themesberg/flowbite-react#1499] [Q&A] [answered]
I'd like to reduce the icon size of DarkThemeToggle, as, when put on the Navbar it makes the links look ugly.
How can this be achived?
![darkthemetogglenavbar](https://github.com/user-attachments/assets/1e1dc44a-befb-4313-b34a-fecd316aec8c)

**Comment by SutuSebastian** [comment, marked as answer]
Use the `theme` prop:

```jsx
<DarkThemeToggle
  theme={{
    root: {
      icon: twMerge(theme.darkThemeToggle.root.icon, "h-3.5 w-3.5"),
    },
  }}
/>
```

Quick tip: 
- link could look good if u center all content `items-center` (in case of using `flex`)

**Reply by idc77** [reply]
Thank you
